content/source/docs/enterprise/before-installing/reference-architecture/azure.md

```diff
@@ -299,44 +325,80 @@ services such as DNS.
-corruption.
+- It can only be run in the _External Services_ mode.
+- The second/alternate and additional nodes are active and processing work at all times.
+- There is an addition to the existing external services which is a memory cache which is currently implemented with cloud native implementations of Redis. This is used for the processing queue for the application and has been moved from the individual instance to be a shared resource that manages distribution of work.
+- There are additional configuration parameters to manage the operation of the node cluster and the memory cache.
 
-##### PostgreSQL Database
+The following sections will provide further detail on the infrastructure and implementation differences.
 
-Backup and recovery of PostgreSQL is managed by Azure and configured
-through the Azure portal or CLI. More details of Azure DB for PostgreSQL
-features are available
-[here](https://docs.microsoft.com/en-us/azure/postgresql/concepts-backup)
-and summarised below:
+### Migration to Active/Active
 
-> *Automated Backups – Azure Database for PostgreSQL automatically
-> creates server backups and stores them in user configured locally
-> redundant or geo-redundant storage.*
->
-> *Backup redundancy – Azure Database for PostgreSQL provides the
-> flexibility to choose between locally redundant or geo-redundant
-> backup storage.*
+If you are considering a migration from a _Standalone_ implementation to _Active/Active_, it is straightforward and there is guidance available to assist with that effort. However, you should first make a determination if the move is necessary. The _Standalone_ mode is capable of handling significant load and the first paths to supporting higher load can be simply increasing the compute power in the existing implementation.  A discussion with your HashiCorp representatives may be warranted.
 
-##### Object Storage
+Also note that if your existing architecture does not already depict what is shown and discussed above, you will likely need to make adjustments to bring it into alignment. This could be either before or during the migration. Certain tenets of the reference architecture described here are highly recommended and potentially necessary to support _Active/Active_ mode such as load balancers and scaling groups.
+
+### Infrastructure Diagram - Active/Active
+
+![aws-aa-infrastructure-diagram](./assets/RA-TFE-AA-Azure-SingleRegion.png)
+
+The above diagram shows the infrastructure components of an _Active/Active_ implementation at a high-level.
+
+### Infrastructure Requirements
+
+#### Active Nodes
+
+The diagram depicts two active nodes to be concise. Additional nodes can be added by simply launching another instance with the identical configuration that points to the same shared external services. The number and sizing of nodes should be based on load requirements and redundancy needs. Nodes should be deployed in alternate zones to accommodate zone failure.
```

Comment:
Not always! Customers using node IP addresses in their configuration will need to make adjustments.

Reply:
altered language but not in detail
All changes reflected in GCP also. Committed under new branch and new PR